Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -134,15 +142,18 @@
   - type: StorageFill
     contents:
       - id: ClothingOuterHardsuitEngineeringWhite
+      - id: OxygenTankFilled
```

Comment:
You're duplicating a oxygen tank right below.

Reply:
resolved